reference/docs-conceptual/components/vscode/understanding-file-encoding.md

```diff
@@ -0,0 +1,320 @@
+  PowerShell expects. This can happen when collaborators use editors with different encoding
+  configurations.
+
+### How to tell when you have encoding issues
+
+Often encoding errors present themselves as parse errors in scripts. If you find strange character
+sequences in your script, this can be the problem. In the example below, an en-dash (`–`) appears as
+the characters `â€“`:
+
+```Output
+Send-MailMessage : A positional parameter cannot be found that accepts argument 'Testing FuseMail SMTP...'.
+At C:\Users\<User>\<OneDrive>\Development\PowerShell\Scripts\Send-EmailUsingSmtpRelay.ps1:6 char:1
++ Send-MailMessage â€“From $from â€“To $recipient1 â€“Subject $subject  ...
++ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : InvalidArgument: (:) [Send-MailMessage], ParameterBindingException
+    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.SendMailMessage
+```
+
+This problem occurs because VSCode encodes the character `–` in UTF-8 as the bytes `0xE2 0x80 0x93`.
+When these bytes are decoded as Windows-1252, they are interpreted as the characters `â€“`.
+
+Some strange character sequences that you might see include:
+
+- `â€“` instead of `–`
+- `â€”` instead of `—`
+- `Ã„2` instead of `Ä`
+- `Â` instead of ` `  (a non-breaking space)
+- `Ã©` instead of `é`
+
+This handy [reference](https://www.i18nqa.com/debug/utf8-debug.html) lists the common patterns that
+indicate a UTF-8/Windows-1252 encoding problem.
+
+## How the PowerShell extension in VSCode interacts with encodings
+
+The PowerShell extension interacts with scripts in a number of ways:
+
+1. When scripts are edited in VSCode, the contents are sent by VSCode to the extension. The [Language Server Protocol][]
+   mandates that this content is transferred in UTF-8. Therefore, it is not possible for the
+   extension to get the wrong encoding.
+2. When scripts are executed directly in the Integrated Console, they're read from the file by
+   PowerShell directly. Tf PowerShell's encoding differs from VSCode's, something can go wrong here.
+3. When a script that is open in VSCode references another script that is not open in VSCode, the
+   extension falls back to loading that script's content from the file system. VSCode defaults to
+   UTF-8 encoding, but uses [byte-order mark][], or BOM, detection to select the correct encoding.
+
+The problem occurs when assuming the encoding of BOM-less formats (like [UTF-8] with no BOM and [Windows-1252]).
+The PowerShell extension defaults to UTF-8. The extension cannot change VSCode's encoding settings.
+For more information, see [issue #824](https://github.com/Microsoft/vscode/issues/824).
+
+## Choosing the right encoding
+
+Choosing an encoding depends on the platforms and applications you use to read and write your
+PowerShell files.
+
+On Windows, many applications have long used [Windows-1252]. Many .NET applications use [UTF-16],
```

Comment:
@joeyaiello made the good point that the `Unicode` term is mainly used in .NET.

It's important because of the `[System.Text.Encoding]::Unicode` enum value and the `Get-Content -Encoding Unicode` parameter value, which both refer to UTF-16. We have to say "Unicode", but don't want people to think they want to use the "Unicode" encoding.

Reply:
Used your bulleted list.